lfs/config.go

```diff
@@ -134,11 +141,10 @@ func (c *Configuration) BatchTransfer() bool {
 		}
 
 		// Any numeric value except 0 is considered true
-		if n, err := strconv.Atoi(v); err == nil && n != 0 {
-			return true
-		}
+		n, _ := strconv.Atoi(v)
+		return n != 0
```

Comment:
I updated the tests to use a matrix so it was easier to see what values were being tested. "false" wasn't specifically tested, but it works. `strconv.Atoi()` returns `0` even if there's an error parsing the string to a number, so it still passes.